src/seedu/addressbook/data/person/Name.java

```diff
@@ -60,8 +60,11 @@ public int hashCode() {
         return fullName.hashCode();
     }
 
+    /**
+     * The comparator now check in lexocographical order and is case in-sensitive
+     */
     @Override
-    public int compareTo(Name name){
+    public int compareTo(Name name) {
         return this.fullName.compareToIgnoreCase(name.fullName);
```

Comment:
Apologies for missing this out earlier: 
If you are doing case-insensitive comparison of strings, perhaps it would be better to mention that the names are sorted using case-insensitive lexicographical order!